### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by cendhu**
Signed-off-by: senthil <cendhu@gmail.com>

### CODE REVIEW

File: docs/getting-started/launching-one-node/docker.md [hunk 1]
```diff
@@ -9,51 +9,79 @@ title: Orion Docker
...
+
+  1. Create the required directory using the command `mkdir -p github.com/hyperledger-labs`
+  2. Change the current working directory to the above created directory by issing the command `cd github.com/hyperledger-labs`
+  3. Clone the server repository with `git clone https://github.com/hyperledger-labs/orion-server`
+  4. Change the current working directory to the repository root directory by issuing `cd orion-server`
+  5. To build the docker image, issue the command `make docker` which will create a docker image named `orionbcdb/orion-server`
+
+## 3) Start Orion in a Docker Container
+
+To start a node, we need a certificate authority and crypto materials for the node and admin users. To simplify this task, we have provided sample
```
Comment (gennadylaventman) [inline docs/getting-started/launching-one-node/docker.md:46]: for the node and admin users -> for the server node and the admin user.

File: docs/getting-started/launching-one-node/docker.md [hunk 2]
```diff
@@ -9,51 +9,79 @@ title: Orion Docker
...
 
-#### Existing volumes
-As mentioned before, by default, the ledger & database data and the server configuration are stored inside the docker container, but it can be overwritten by mapping host folders to container volumes.
+Port `6001` is Orion REST API port and should be used to access Orion using its REST API or [Go SDK](https://github.com/hyperledger-labs/orion-sdk-go/)
+
+Port `7050` is used between nodes in a cluster for performing replication.
+
+:::info
+To change the port `6001` and `7050`, we need to change the configuration file and map the host directory holding this configuration to `/etc/orion-server/config`
+as explained next in [volumes](#volumes-for-pesistant-storage,-configuration,-and-crypto-materials).
```
Comment (gennadylaventman) [inline docs/getting-started/launching-one-node/docker.md:62]: I didn't got how internal links are formed...

File: docs/getting-started/launching-one-node/docker.md (hunk 1 above)
Comment (cendhu) [reply docs/getting-started/launching-one-node/docker.md]: Done

File: docs/getting-started/launching-one-node/docker.md (hunk 2 above)
Comment (cendhu) [reply docs/getting-started/launching-one-node/docker.md]: Actually, I had used a wrong link. Now, I have updated it.

**Review by gennadylaventman — APPROVED** [review]

**Pull request merged**